frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,36 @@
+import { IconName } from '~/libs/enums/icon-name.enum';
```

Comment:
```suggestion
import { IconName } from '~/libs/enums/icon-name.enum.js';
```
Please add extension, we'll investigate why linter missing this check